packages/react-instantsearch/src/connectors/connectInfiniteHits.js

```diff
@@ -5,6 +5,10 @@ import {PropTypes} from 'react';
 
 import createConnector from '../core/createConnector';
 
+function getId() {
+  return 'pager';
```

Comment:
True, will update that to page. I will make that consistent with the pagination then.